src/event.rs

```diff
@@ -232,26 +232,58 @@ where
     reader.read(filter)
 }
 
+/// Mouse capture mode
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum MouseTrackingMode {
```

Comment:
Maybe something like a bitmask (attributes) (already have that in the deps) work here?